Link Configuration Files in GitHub

lkiesow opened the pull request:
This patch automatically links configuration files referenced in the
documentation to the actual configuration files in GitHub. This is done
in addition to the tooltip added to `etc/` explaining where to find
these files.


![Screenshot from 2021-12-03 14-31-32](https://user-images.githubusercontent.com/1008395/144611171-8f32c7a3-068b-4416-b61c-366e91e2ba62.png)

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

lkiesow committed:
Link Configuration Files in GitHub

This patch automatically links configuration files referenced in the
documentation to the actual configuration files in GitHub. This is done
in addition to the tooltip added to `etc/` explaining where to find
these files.

ziegenberg commented on docs/guides/admin/docs/javascript/extra.js:
```diff
@@ -50,12 +50,30 @@ function addTitleToCodeTag() {
+
+                // Link repository
+                let a = document.createElement('a');
+                a.innerText = '⇲';
```
Instead of the rotated icon ⇲ (aka South East Arrow to Corner `&#8690;`), one could use 📝 (`&#128221;`) or 📄 (`&#128196;`).

gregorydlogan replied:
I feel like those aren't actually any better though.  At least to my eyes, the 128221 reads as 'edit this' and the other is just a gray blob.

mliradelc replied:
In that case, a gearbox should be better ⚙️, but I like the arrow shown in the example.  The magnify glass also should work 🔍.

lkiesow commented:
Before we continue down the rabbit hole of what arbitrary character could we misuse for this, if you feel strongly about not using the arrow, we could just use the actual GitHub logo instead:

![Screenshot from 2021-12-15 01-56-51](https://user-images.githubusercontent.com/1008395/146103347-5334028c-aaf2-43eb-854b-cdba4ff81c3e.png)

What do you think?

gregorydlogan commented:
This seems like a good solution to me. Makes it clear it's offsite, and on gh.

lkiesow committed:
Switch to GitHub icon for documentation link

This patch switches to the actual GitHub icon for links to configuration
file in the git repository on docs.opencast.org.

lkiesow commented:
Since I liked the result, I already included this  in this pull request yesterday. Happy to get a review for this.
